README.md

```diff
@@ -23,6 +23,9 @@ $ composer require accompli/accompli --dev
 #### Versioning
 Accompli uses [Semantic Versioning 2](http://semver.org/) for new versions.
 
+#### Documentation
+View the complete documentation of Accompli on [accompli.readme.io](https://accompli.readme.io/).
```

Comment:
Please move this line above the `Installation using Composer` header (without it's own header).